README.md

```diff
@@ -1,2 +1,69 @@
-# cap-js
-An SPA web-app based on vanilla javascript to fetch items and add likes and comments to another store API
+# Microverse JS Capstone Project
+
+> A Single Page Web App that posts and gets data from different APIs to list some tv-shows. As well it the SPA has the functionality to post likes and comments to the listed tv-shows.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Git
+- Webpack
+- Jest
+- Babel
+
+## Live Demo
+
+[Live Demo Link](https://hsztan.github.io/cap-js/)
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
```

Comment:
- [ ] I would recommend adding the video presentation link which includes the following:

1. presenting the features of the project.
1. The maximum length of the video is 5 minutes.
1. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
1. All the members in the video should appear in the video and talk about the project.
1. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.